src/main/java/utils/Utils.java

```diff
@@ -0,0 +1,20 @@
+package utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Utils {
+    public static void waitInSeconds (int seconds) {
+    try {
+        Thread.sleep(seconds * 1000);
+    }
+    catch (Exception e){
+        System.err.println(e.getMessage());
+    }
```

Comment:
The try-catch block does not respect code indentation. Try formatting all code lines.

Reply:
done